http_requests.py

```diff
@@ -75,40 +77,70 @@ def mount_default_adapter(
     return session
 
 
+def _create_logging_client():
+    '''Create an ELS Client for the default els in the active config set and return it
```

Comment:
nit: the first sentence is not correct. and it should not be correct. and thus the name is not well-chosen.

what this function _should_ do is return (and if necessary create beforehand) the singleton instance of the logging client or `None` if no such client is available (e.g. because execution environment is not central CI/CD).

on the other hand, this function - albeit private - should not be used by anyone but the consumer thread anyways, so I do not consider it a good idea to implement it in the first place

starting to look in more detail in this PR. why is this code added to `http_request.py`? this is not the right place at all! the asynchronous handling should of course be done (in a transparent manner) in the elasticsearch client (`ccc.elasticsearch`, I think?)